setup.env

```diff
@@ -31,7 +31,7 @@ $ONL/tools/make-versions.py --import-file=$ONL/tools/onlvi --class-name=OnlVersi
 #
 # buildroot download mirror. We suggest you setup a local repository containing these contents for faster local builds.
 #
-export BUILDROOTMIRROR=${BUILDROOTMIRROR:-"http://buildroot.opennetlinux.org/dl"}
```

Comment:
`buildroot.opennetlinux.org/dl` is no longer available(host is down)
There is a GitHub repo that holds all files: https://github.com/opennetworklinux/buildroot-download-cache

Reply:
We actually not using the buildroot tool to build the initrd image, but use prebuilt one from [sm/build-artifacts](https://github.com/opennetworklinux/build-artifacts)